src/h.js

```diff
@@ -912,6 +912,14 @@ const
             scripts[i].parentNode.removeChild(scripts[i]);
         }
         return div.innerHTML;
+    },
+    sleep = (milliseconds: integer) => {
```

Comment:
I think this will hang the whole thread while being executed. Maybe we should use something like this https://stackoverflow.com/a/39914235

Reply:
Good catch! Will update it. :)